src/index.js

```diff
@@ -80,8 +80,23 @@ exports.ensureAuthenticated = (request, response, next) => {
 
 let alreadyInitialized = false;
 
+exports.initServices = (Implementation) => {
```

Comment:
Is it necessary to expose (`exports`) this function?

Reply:
fixed